src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_ldm3d.py

```diff
@@ -49,7 +49,7 @@
         >>> pipe = pipe.to("cuda")
 
         >>> prompt = "a photo of an astronaut riding a horse on mars"
-        >>> output = pipe_ldm3d(prompt)
+        >>> output = pipe(prompt)
```

Comment:
Would you maybe also like to fix the Python code snippet in the doc? 
<img width="1049" alt="image" src="https://github.com/huggingface/diffusers/assets/22957388/4a377914-1767-460b-9b93-985422928628">

As we can see the snippet doesn't properly reflect the Python interpreter prompt (`>>>`). We can either get rid of that or ensure all the lines have that.

Reply:
Sure Ill update the doc accordingly :)